e2e-tests/src/page-components/question.ts

```diff
@@ -0,0 +1,110 @@
+import { Locator, Page, expect } from '@playwright/test'
+import { PageComponentInterface } from 'src/models/page-component-interface'
+import Radiogroup from 'src/page-components/radiogroup'
+import Textbox from 'src/page-components/textbox'
+
+export default class Question implements PageComponentInterface {
```

Comment:
`Question` is the class that can be used to find everything inside a question in study registration workflow. For example,  is this a required question? Is there any error and what the error is? What's the question title? How to click to answer this question? etc.